<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPythonEdit" name="pythonEdit" native="true"/>
       <widget class="GLPreviewWidget" name="glPreviewWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
      <widget class="QTextEdit" name="consoleTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Files">
     <property name="title">
      <string>Recent Files</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionClear_All"/>
    </widget>
    <addaction name="actionNew_File"/>
    <addaction name="actionOpen_File"/>
    <addaction name="menuRecent_Files"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menuFind_Replace">
     <property name="title">
      <string>&amp;Find/Replace</string>
     </property>
     <addaction name="actionFind_Replace"/>
     <addaction name="separator"/>
     <addaction name="actionFind_Next"/>
     <addaction name="actionFind_Previous"/>
     <addaction name="actionReplace"/>
     <addaction name="actionReplace_All"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="menuFind_Replace"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionFull_Screen"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>&amp;Run</string>
    </property>
    <addaction name="actionRun_From_Start"/>
    <addaction name="actionRun_From_Last_Statement"/>
   </widget>
   <widget class="QMenu" name="menu3D_View">
    <property name="title">
     <string>3D View</string>
    </property>
    <addaction name="actionPerspective"/>
    <addaction name="actionReset_Camera"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuRun"/>
   <addaction name="menuWindow"/>
   <addaction name="menu3D_View"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="findAndReplacedockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="windowTitle">
    <string>Find And Replace</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Find</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="findTextEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Replace</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="replaceTextEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="findButton">
           <property name="text">
            <string>&amp;Find</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="replaceButton">
           <property name="text">
            <string>&amp;Replace</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="replaceAllButton">
           <property name="text">
            <string>Replace &amp;All</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="findCaseSensitiveCheckBox">
           <property name="text">
            <string>Case Sensitive</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="findRegexCheckBox">
           <property name="text">
            <string>Regex</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="findWholeWordCheckBox">
           <property name="text">
            <string>Whole word</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew_File">
   <property name="text">
    <string>&amp;New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>&amp;Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_all">
   <property name="text">
    <string>Save a&amp;ll</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="text">
    <string>Close All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>&amp;Undo </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select &amp;All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F11</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionClear_All">
   <property name="text">
    <string>Clear All</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionFind_Replace">
   <property name="text">
    <string>Find/Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFind_Next">
   <property name="text">
    <string>Find Next</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionFind_Previous">
   <property name="text">
    <string>Find Previous</string>
   </property>
   <property name="shortcut">
    <string>Shift+F3</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>Replace</string>
   </property>
  </action>
  <action name="actionReplace_Find">
   <property name="text">
    <string>Replace &amp;&amp; Find</string>
   </property>
  </action>
  <action name="actionReplace_All">
   <property name="text">
    <string>Replace All</string>
   </property>
  </action>
  <action name="actionForce_Update">
   <property name="text">
    <string>Force Update</string>
   </property>
  </action>
  <action name="actionRun_From_Start">
   <property name="text">
    <string>Run From &amp;Start</string>
   </property>
   <property name="iconText">
    <string>Run From Start</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRun_From_Last_Statement">
   <property name="text">
    <string>Run From &amp;Last Statement</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionPerspective">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Ortho/Perspective</string>
   </property>
  </action>
  <action name="actionReset_Camera">
   <property name="text">
    <string>Reset Camera</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPythonEdit</class>
   <extends>QWidget</extends>
   <header>qpython_edit</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GLPreviewWidget</class>
   <extends>QWidget</extends>
   <header>gl_preview_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
